common/game-state-queue.cpp

```diff
@@ -112,30 +112,57 @@ namespace tec {
 		for (int i = 0; i < entity.components_size(); ++i) {
 			const proto::Component& comp = entity.components(i);
 			switch (comp.component_case()) {
-			case proto::Component::kPosition:
-			{
-				Position pos;
-				pos.In(comp);
-				this->interpolated_state.positions[entity_id] = pos;
-				this->base_state.positions[entity_id] = pos;
-			}
-			break;
-			case proto::Component::kOrientation:
-			{
-				Orientation orientation;
-				orientation.In(comp);
-				this->interpolated_state.orientations[entity_id] = orientation;
-				this->base_state.orientations[entity_id] = orientation;
-			}
-			break;
-			case proto::Component::kVelocity:
-			{
-				Velocity vel;
-				vel.In(comp);
-				this->interpolated_state.velocities[entity_id] = vel;
-				this->base_state.velocities[entity_id] = vel;
-			}
-			break;
+				case proto::Component::kPosition:
+				{
+					Position pos;
+					pos.In(comp);
+					this->interpolated_state.positions[entity_id] = pos;
+					this->base_state.positions[entity_id] = pos;
+				}
+					break;
+				case proto::Component::kOrientation:
+				{
+					Orientation orientation;
+					orientation.In(comp);
+					this->interpolated_state.orientations[entity_id] = orientation;
+					this->base_state.orientations[entity_id] = orientation;
+				}
+					break;
+				case proto::Component::kVelocity:
+				{
+					Velocity vel;
+					vel.In(comp);
+					this->interpolated_state.velocities[entity_id] = vel;
+					this->base_state.velocities[entity_id] = vel;
+				}
+					break;
+				case proto::Component::kRenderable:
+					break;
```

Comment:
Most of these breaks can be removed as the cases will falldown till exit regardless.